release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1166,6 +1166,19 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 ==== Filtering custom resources during ZTP spoke cluster installation using SiteConfig filters
 You can now use filters to customize `SiteConfig` CRs to include or exclude other CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline. For more information, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-filtering-ai-crs-using-siteconfig_ztp-deploying-disconnected[Filtering custom resources using SiteConfig filters].
 
+[id="ocp-4-10-RAN-disabling-chronyd"]
+====  Disable chronyd in the PolicyGenTemplate CR
+
+You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the group `PolicyGenTemplate` CR:
```

Comment:
Just a minor nit based on @tliu2021's inputs, would the following be more appropriate?
```suggestion
You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is referenced in the group `PolicyGenTemplate` CR:
```